### ISSUE TITLE
Document adding a timestamp column with SQL `NOW()` as default value

### ISSUE THREAD

**Issue by notramo** [luckyframework/avram#944]
Is it possible to add a timestamp typed column to an existing table, with the default value being a raw SQL `NOW()`? So if a new record is added, the column value is set to the timestamp the column was added.

**Comment by jwoertink** [comment]
Yup. `add time : Time, default: :now`. Looks like it's just not documented.
